sdk/eventhub/event-hubs/src/eventDataBatch.ts

```diff
@@ -6,6 +6,35 @@ import { ConnectionContext } from "./connectionContext";
 import { AmqpMessage } from "@azure/core-amqp";
 import { message } from "rhea-promise";
 import { throwTypeErrorIfParameterMissing } from "./util/error";
+import { Span, SpanContext } from "@azure/core-tracing";
+import { instrumentEventData, TRACEPARENT_PROPERTY } from "./diagnostics/instrumentEventData";
+import { createMessageSpan } from "./diagnostics/messageSpan";
+
+/**
+ * Checks if the provided eventDataBatch is an instance of `EventDataBatch`.
+ * @param eventDataBatch The instance of `EventDataBatch` to verify.
+ * @internal
+ * @ignore
+ */
+export function isEventDataBatch(eventDataBatch: any): eventDataBatch is EventDataBatch {
+  return (
+    eventDataBatch &&
+    typeof eventDataBatch.tryAdd === "function" &&
+    typeof eventDataBatch.count === "number" &&
+    typeof eventDataBatch.sizeInBytes === "number"
+  );
+}
+
+/**
+ * The set of options to configure the behavior of `tryAdd`.
+ * - `parentSpan` : The `Span` or `SpanContext` to use as the `parent` of the span created while calling this operation.
+ */
+export interface TryAddOptions {
```

Comment:
this same interface is defined in messageSpan.ts

Reply:
It does not appear to be so.